CMakeLists.txt

```diff
@@ -551,6 +552,66 @@ if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/nnvm/CMakeLists.txt")
   list(APPEND mxnet_LINKER_LIBS ${nnvm_LINKER_LIBS})
 endif()
 
+# ---[ NNPack
+if(USE_NNPACK)
+  # Add in NNPack and its dependencies
+  set(NNPACK_ROOT_DIR "${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/nnpack")
+  if (EXISTS "${NNPACK_ROOT_DIR}" AND EXISTS "${NNPACK_ROOT_DIR}/NNPACK")
+    FIND_PACKAGE(PythonInterp)
+    IF(NOT PYTHONINTERP_FOUND)
+      MESSAGE(STATUS "Python interpreter not found; setting Python version to 0.0.0")
+      SET(PYTHON_VERSION_STRING "0.0.0")
```

Comment:
If Python interpreter not found, you also don't need to setup Python packages (PeachPy, enum34, and six)